frontend/src/metabase/public/containers/PublicQuestion/PublicQuestion.unit.spec.tsx

```diff
@@ -87,12 +91,16 @@ describe("PublicQuestion", () => {
   it("should update card settings when visualization component changes them (metabase#37429)", async () => {
     await setup();
 
+    await waitForLoaderToBeRemoved();
```

Comment:
Jest tests were failing without these. Considering we are using a `LoadingAndErrorWrapper` I'm not particularly sure how this worked in the first place without these `waitForLoaderToBeRemoved` calls